packages/requestNetworkSmartContracts/contracts/test/vendor/KyberMock.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity 0.4.18;
+
+
+/// @dev From https://github.com/KyberNetwork/smart-contracts/blob/master/contracts/ERC20Interface.sol
+interface ERC20 {
```

Comment:
Could be imported (used in two contracts: KyberMock & Burner)

Reply:
Discussed offline. The two are slightly different. The ERC20 in Burner has an additional burn() function